### PR TITLE
Feature dtcenter/METplus-Internal#15 add checksum to release

### PR DESCRIPTION
**Pull request by georgemccabe**
GitHub Issue: dtcenter/METplus-Internal#15

## Expected Differences ##

- [X] Do these changes introduce new tools, command line arguments, or configuration file options? **[No]**</br>

- [X] Do these changes modify the structure of existing or add new output data types (e.g. statistic line types or NetCDF variables)? **[No]**</br>

## Pull Request Testing ##

- [X] Describe testing already performed for these changes:</br>

Created a test release to ensure that workflow ran and added checksum files to release

- [X] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>

Review documentation changes

- [X] Do these changes include sufficient documentation updates, ensuring that no errors or warnings exist in the build of the documentation? **[Yes]**

- [X] Do these changes include sufficient testing updates? **[Yes]**

- [X] Will this PR result in changes to the test suite? **[No]**</br>

- [X] Please complete this pull request review by **9/13/2022**.</br>

## Pull Request Checklist ##
See the [METplus Workflow](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html) for details.
- [X] Review the source issue metadata (required labels, projects, and milestone).
- [X] Complete the PR definition above.
- [X] Ensure the PR title matches the feature or bugfix branch name.
- [X] Define the PR metadata, as permissions allow.
Select: **Reviewer(s)**
Select: **Organization** level software support **Project** or **Repository** level development cycle **Project**
Select: **Milestone** as the version that will include these changes
- [X] After submitting the PR, select **Development** with the original issue number. issue linked above
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue and delete your feature or bugfix branch from GitHub.

### CODE REVIEW

File: docs/Users_Guide/installation.rst [hunk 1]
```diff
@@ -143,8 +143,32 @@ Building the MET Package
...
+    - Run the *sha256sum* command with the --check argument to verify that the source code download file was not corrupted.
+
+Zip File::
+
+    sha256sum --check checksum_zip.txt
+
+Tar File::
+
+    sha256sum --check checksum_tar.txt
+
```
Comment (jprestop) [inline docs/Users_Guide/installation.rst:167]:
```suggestion

.. note::
   If the source code is downloaded using **wget**, then the filenames will not
   match the filenames listed in the checksum files. If the source code is
   downloaded using **curl**, the *-LJO* flags should be added to the command to
   preserve the expected filenames found in the checksum files.
   
```

**Review by jprestop — CHANGES_REQUESTED** [review]
I reviewed the documentation and added a suggestion the addition of the note about wget and curl.

**Comment by georgemccabe** [comment]
Thanks, Julie. I agree we that should add this note.

> I reviewed the documentation and added a suggestion the addition of the note about wget and curl.

**Review by jprestop — APPROVED** [review]
Thanks, @georgemccabe. All checks have passed.  I approve this request.

**Comment by JohnHalleyGotway** [comment]
I am proceeding with merging this PR on 9/14/2022 based on the review approval by @jprestop to make sure that it's included in the met-11.0.0-beta3 release.

**Pull request merged**